rpc/README.md

```diff
@@ -3512,6 +3523,19 @@ In JSONRPC, it is encoded as a 0x-prefixed hex string.
 
 
 
+### Type `HardForkFeature`
+
+The information about one hardfork feature.
+
+#### Fields
+
+`HardForkFeature` is a JSON object with the following fields.
+
+*   `rfc`: `string` - The related RFC ID.
+
+*   `epoch`: [`EpochNumber`](#type-epochnumber) `|` `null` - The first epoch when the feature is enabled.
```

Comment:
Please explain what does `null` mean.

Reply:
previously I misunderstood *epoch* as `EpochNumberWithFraction`, while it should actually be interpreted as `EpochNumber`. And sadly the underlying types of  `EpochNumberWithFraction` and `EpochNumber` are both `u64`.

I would suggest changing the name to *epoch_number*, to reduce ambiguity.